packages/scripts/src/commands/app-data/convert/processors/flowParser/parsers/default.parser.ts

```diff
@@ -285,6 +287,11 @@ class RowProcessor {
       if (typeof this.row[field] === "string") {
         // remove whitespace
         this.row[field] = this.row[field].trim();
+
+        // remove all line breaks in the field using replaceAll() and regex
```

Comment:
This comment doesn't describe what the logic below does (and we wouldn't want to remove all line breaks). It should say something like:
"replace any strings that consist only of whitespace with the empty string"